test/amp_tags_test.js

```diff
@@ -85,4 +135,16 @@ describe('AmpTags', function () {
     );
     assertContains('<link rel="amphtml" href="https://madrid.quintype.io/amp/story/section/slug"/>', string);
   });
+
+  it("does encode the visual story url if encodeAmpUrl is set to false in seoConfig", function () {
+    const story = { slug: "section%2Fslug", "is-amp-supported": true };
+    const string = getSeoMetadata(
+      { ...seoConfig, appendHostToAmpUrl: true, decodeAmpUrl: true },
+      config,
+      "story-page",
+      { currentHostUrl: "https://madrid.quintype.io/section%2Fslug", data: { story: { ...story, "story-template": "visual-story" } } },
+      {}
+    );
+    assertContains('<link rel="amphtml" href="https://madrid.quintype.io/section/slug"/>', string);
+  });
```

Comment:
independent of story template, test is not needed

Reply:
removed